src/azure-cli/azure/cli/command_modules/rdbms/validators.py

```diff
@@ -442,10 +442,11 @@ def ip_address_validator(ns):
 def public_access_validator(ns):
     if ns.public_access:
         val = ns.public_access.lower()
-        if not (val == 'all' or val == 'none' or (len(val.split('-')) == 1 and _validate_ip(val)) or
+        if not (ns.public_access == 'Disabled' or ns.public_access == 'Enabled' or
+                val == 'all' or val == 'none' or (len(val.split('-')) == 1 and _validate_ip(val)) or
                 (len(val.split('-')) == 2 and _validate_ip(val))):
             raise CLIError('incorrect usage: --public-access. '
-                           'Acceptable values are \'all\', \'none\',\'<startIP>\' and '
+                           'Acceptable values are \'Disabled\', \'Enabled\', \'all\', \'none\',\'<startIP>\' and '
```

Comment:
Do you still want to modify the original validator after splitting the arg type?

Reply:
Yes, We need support 'Enabled' and 'Disabled' in create command.